Packs/Active_Directory_Query/Integrations/Active_Directory_Query/README.md

```diff
@@ -578,6 +629,10 @@ Modifies the user organizational unit within a domain.
 #### Base Command
 
 `ad-modify-user-ou`
+
+##### Required Permissions
+Requires `Read userAccountControl` and `write userAccountControl` permissions.
```

Comment:
Please check this - might be even better
https://learn.microsoft.com/en-us/answers/questions/973272/delegate-help-desk-users-permission-to-move-users